SPECS/containerized-data-importer/containerized-data-importer.spec

```diff
@@ -0,0 +1,449 @@
+
+%package        api
+Summary:        CDI API server
+Group:          System/Packages
+
+%description    api
+The containerized-data-importer-api package provides the kubernetes API extension for CDI
+
+%package        cloner
+Summary:        Cloner for host assisted cloning
+Group:          System/Packages
+
+%description    cloner
+Source and Target cloner image for host assisted cloning
+
+%package        controller
+Summary:        Controller for the data fetching service
+Group:          System/Packages
+
+%description    controller
+Controller for the data fetching service for VM container images
+
+%package        importer
+Summary:        Data fetching service
+Group:          System/Packages
+
+%description    importer
+Data fetching service for VM container imagess
+
+%package        operator
+Summary:        Operator for the data fetching service
+Group:          System/Packages
+
+%description    operator
+Operator for the data fetching service for VM container images
+
+%package        uploadproxy
+Summary:        Upload proxy for the data fetching service
+Group:          System/Packages
+
+%description    uploadproxy
+Upload proxy for the data fetching service for VM container images
+
+%package        uploadserver
+Summary:        Upload server for the data fetching service
+Group:          System/Packages
+
+%description    uploadserver
+Upload server for the data fetching service for VM container images
+
+%package        manifests
+Summary:        YAML manifests used to install CDI
+Group:          System/Packages
+
+%description    manifests
+This contains the built YAML manifests used to install CDI into a
+kubernetes installation with kubectl apply.
+
+%prep
+# Unpack the sources respecting the GOPATH directory structure expected by the
+# go imports resolver. I.e. if DIR is in GOPATH then DIR/src/foo/bar can be
+# imported as "foo/bar". The same 'visibility' rules apply to the local copies
+# of external dependencies placed in 'vendor' directory when imported from the
+# 'parent' package.
+#
+# Note: having bar symlink'ed to DIR/src/foo/bar does not seem to work. Looks
+# like symlinks in go path are not resolved correctly. Hence the sources need
+# to be 'physically' placed into the proper location.
+%setup -q -n go/src/kubevirt.io/%{name} -c -T
+tar --strip-components=1 -xf %{SOURCE0}
+
+%build
+# Hackery to determine which registry path to use in cdi-operator.yaml
+# when building the manifests
+
+export GOPATH=%{_builddir}/go
+export GOFLAGS="-buildmode=pie -mod=vendor"
+env \
+CDI_SOURCE_DATE_EPOCH="$(date -r LICENSE +%s)" \
+CDI_GIT_COMMIT='v%{version}' \
+CDI_GIT_VERSION='v%{version}' \
+CDI_GIT_TREE_STATE="clean" \
+./hack/build/build-go.sh build \
+	cmd/cdi-apiserver \
+	cmd/cdi-cloner \
+	cmd/cdi-controller \
+	cmd/cdi-importer \
+	cmd/cdi-uploadproxy \
+	cmd/cdi-uploadserver \
+	cmd/cdi-operator \
+	%{nil}
+
+env DOCKER_PREFIX=$reg_path DOCKER_TAG=%{version}-%{release} ./hack/build/build-manifests.sh
```

Comment:
reg_path is not defined. Do we really need the manifests? If not, then we can remove this.

Reply:
You're right! This too has been removed.  But build-manifests.sh has been retained.